src/support.h

```diff
@@ -4,6 +4,9 @@
 #ifndef SUPPORT_H
 #define SUPPORT_H
 
+#include <memory>
```

Comment:
Why is this necessary? You don't seem to use it here.

Reply:
Remnant of an old test, will remove it